includes/modules_helper.php

```diff
@@ -188,4 +188,53 @@ public function store_left_right($key)
 
 		$this->config->set($key, $value);
 	}
+
+    /**
+     * Show available styles
+     *
+     * @return string Select with available styles
+     */
+    public function display_fa_styles()
+    {
+        global $phpbb_container;
```

Comment:
Don't use globals but instead extend the modules_helper class to have the needed objects passed into the constructor.